src/common/VersionHelper.cpp

```diff
@@ -30,33 +30,33 @@ VersionHelper::VersionHelper(int major, int minor, int revision) :
 
 bool VersionHelper::operator>(const VersionHelper& rhs)
 {
-    if (major < rhs.major)
+    if (major > rhs.major)
```

Comment:
`return std::tie(major, minor, revision) > std::tie(rhs.major, rhs.minor, rhs.revision)`?

By the way, all STL C++ libraries expect `operator<`.